db/seeds.rb

```diff
@@ -50,29 +69,39 @@ def random_record_for_org(org, klass)
   org.items.where(value_in_cents: 0).limit(10).update_all(value_in_cents: 100)
 end
 
+
+# ----------------------------------------------------------------------------
+# Users
+# ----------------------------------------------------------------------------
+
 [
   { email: 'superadmin@example.com', organization_admin: false, super_admin: true },
-  { email: 'org_admin1@example.com', organization_admin: true, organization: pdx_org },
-  { email: 'org_admin2@example.com', organization_admin: true, organization: sf_org },
-  { email: 'user_1@example.com', organization_admin: false, organization: pdx_org },
-  { email: 'user_2@example.com', organization_admin: false, organization: sf_org },
-  { email: 'test@example.com', organization_admin: false, super_admin: true, organization: pdx_org },
-  { email: 'test2@example.com', organization_admin: true, organization: pdx_org }
+  { email: 'org_admin1@example.com', organization_admin: true,                     organization: pdx_org },
+  { email: 'org_admin2@example.com', organization_admin: true,                     organization: sf_org },
+  { email: 'user_1@example.com',     organization_admin: false,                    organization: pdx_org },
+  { email: 'user_2@example.com',     organization_admin: false,                    organization: sf_org },
+  { email: 'test@example.com',       organization_admin: false, super_admin: true, organization: pdx_org },
```

Comment:
It's moot and not a blocker, but if you swapped the order of `super_admin` and `organization` (since these are hashes), you woudn't have to do all that whitespace there :)

Reply:
I figured that the top line should "win". If not, there would still be a lot of whitespace in the first line, but it's true that there would be less in several other lines. I considered swapping lines but figured I should leave them as is. Anyway, I've changed this and pushed the change.